lib/docker/docker_image.js

```diff
@@ -86,13 +86,17 @@ export default class DockerImage {
       pullOptions.authconfig = this.credentials;
     }
 
-    return await this.runtime.stats.timeGen(
-        'dockerImageDownloadTime',
-        this.pullImageStreamTo(this.runtime.docker,
-                               dockerImageName,
-                               this.stream,
-                               pullOptions)
+    let start = Date.now();
```

Comment:
yea this is pretty much what I was doing here [1] but decided to just rip it all out for now.  I don't do measurements too much in the worker so I took the easy way out.  I was going to port over this stat wrapper to use the new monitor client but just seemed to be unnecessary until the client itself supports a timed handler of sorts.

[1] https://github.com/taskcluster/docker-worker/blob/master/lib/stats/stat.js#L94